python/lsst/obs/monocam/monocamIsrTask.py

```diff
@@ -84,6 +86,10 @@ def run(self, ccdExposure, bias=None, dark=None,  flat=None, defects=None, fring
                 ampExposure = ccdExposure.Factory(ccdExposure, amp.getBBox())
                 self.updateVariance(ampExposure, amp)
 
+        # Don't trust the variance not to be negative (over-subtraction of dark?)
+        variance = ccdExposure.getMaskedImage().getVariance().getArray()
+        variance[:] = numpy.where(variance > 0, variance, 100.0)
```

Comment:
How is the value 100 motivated?  Seems like it could be configurable or calculated from the data.